testsuite/tests/tool-toplevel-invocation/Makefile

```diff
@@ -0,0 +1,37 @@
+#**************************************************************************
+#*                                                                        *
+#*                                OCaml                                   *
+#*                                                                        *
+#*                          Bernhard Schommer                             *
+#*                                                                        *
+#*   Copyright 2010 Institut National de Recherche en Informatique et     *
+#*     en Automatique.                                                    *
+#*                                                                        *
+#*   All rights reserved.  This file is distributed under the terms of    *
+#*   the GNU Lesser General Public License version 2.1, with the          *
+#*   special exception on linking described in the file LICENSE.          *
+#*                                                                        *
+#**************************************************************************
+
+BASEDIR=../..
+
+
+default:
+	@for file in *.txt; do \
+	 TERM=dumb $(OCAML) -args $$file < test.ml 2>&1 \
+	    | grep -v '^        OCaml version' \
+	    | grep -v ' Script file is not allowed in expanded option.'  > $$file.result; \
```

Comment:
could you turn this into `>> $$file.result`, with an `echo "Script file is not allowed in expanded option." > $$file.result; \` before that? It is a bit silly, but it help users make sense of what the test is testing by looking at the result file, without having to look at the Makefile.